DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -83,7 +83,7 @@ GROUP BY
                         AND Sections.SectionID = @sectionId
                         AND (Sections.ArchivedDate IS NULL)
                         AND (CustomisationTutorials.DiagStatus = 1 OR Customisations.IsAssessed = 1 OR CustomisationTutorials.Status = 1)
-                    ORDER BY Tutorials.OrderByNumber",
+                    ORDER BY Tutorials.OrderByNumber, Tutorials.TutorialID",
```

Comment:
Is there a test we can add for this? And this probably needs to be done on the initial menu page as well, for the sections? If so @benjimarshall it probably makes sense for you to do that as you're changing that page? 🙂

Reply:
@benjimarshall? 🙂

Reply:
Yep I'll add that to HEEDLS-230. Sorry for missing these comments 🙂

Reply:
Great, thanks 🙂